EasyLayout.Forms/EasyLayout.cs

```diff
@@ -491,32 +495,14 @@ private static RightExpression ParseRightExpression(Expression expr, RelativeLay
             }
             else
             {
-                var fExpr = expr as MethodCallExpression;
-                if (fExpr != null)
+                if (expr is MethodCallExpression fExpr)
                 {
                     position = GetPosition(fExpr);
                     memberExpression = fExpr.Arguments.FirstOrDefault() as MemberExpression;
                 }
             }
 
-            if (position == null)
```

Comment:
I put a lot of time into providing helpful error messages for when people use the api wrong.  Is there any way we can get this back in?